src/databricks/sqlalchemy/_types.py

```diff
@@ -33,6 +33,7 @@ def process_literal_param_hack(value: Any):
 @compiles(sqlalchemy.types.Unicode, "databricks")
 @compiles(sqlalchemy.types.UnicodeText, "databricks")
 @compiles(sqlalchemy.types.Uuid, "databricks")
+@compiles(sqlalchemy.types.Json, "databricks")
```

Comment:
We should add a test to cover the new type? I think we have existing test covering other types already.